src/almanack/metrics/data.py

```diff
@@ -56,8 +58,12 @@ def get_table(repo_path: str) -> List[Dict[str, Any]]:
     computation, an exception is raised.
 
     Args:
-        repo_path (str): The file path to the repository for which metrics are
-        to be performed.
+        repo_path (str):
+            The file path to the repository for which metrics are
+            to be performed.
```

Comment:
```suggestion
            The file path to the repository for which the Almanack runs metrics.
```

Switch to active voice - is my suggestion accurate?

Reply:
Thanks, yes!